packages/base-types/src/models/project/index.ts

```diff
@@ -36,6 +36,7 @@ export interface Model<PlatformData extends AnyRecord, MemberPlatformData extend
   type?: string; // chat | voice | etc
   image?: string;
   privacy?: Privacy;
+  APIprivacy?: Privacy;
```

Comment:
Why such weird casing?

```suggestion
  apiPrivacy?: Privacy;
```